config/orte_check_lsf.m4

```diff
@@ -120,6 +120,71 @@ AC_DEFUN([ORTE_CHECK_LSF],[
+          # have failed with messages like:
+          #   lib64/libevent_pthreads.so: undefined reference to `evthread_set_condition_callbacks'
+          #   lib64/libevent_pthreads.so: undefined reference to `event_mm_malloc_'
+          #   lib64/libevent_pthreads.so: undefined reference to `event_mm_free_'
+          #   lib64/libevent_pthreads.so: undefined reference to `evthread_set_id_callback'
+          #   lib64/libevent_pthreads.so: undefined reference to `evthread_set_lock_callbacks'
+          # Because it picked up -levent from LSF, but -levent_pthreads from Libevent.
+          #
+          # So look for a function that libevent_pthreads is looking for from libevent.so
+          # If it does not appear here when adding the same option then we have
+          # a conflict.
+          orte_check_lsf_event_conflict=na
+          AS_IF([test "$orte_check_lsf_happy" = "no"],
+                [AS_IF([test "$opal_event_external_support" = "yes"],
+                       [AS_IF([test "$orte_check_lsf_libdir" = "" ],
+                              [],
+                              [LDFLAGS="$LDFLAGS -L"$orte_check_lsf_libdir])
+                        # Note that we do not want to set LIBS here to include -llsf since
+                        # the check is not for an LSF library, but for the conflict with
+                        # LDFLAGS.
+                        AC_CHECK_LIB([event], [evthread_set_condition_callbacks],
+                                     [AC_MSG_CHECKING([for libevent conflict])
+                                      AC_MSG_RESULT([No. The correct libevent.so was linked.])
+                                      orte_check_lsf_event_conflict=no],
+                                     [AC_MSG_CHECKING([for libevent conflict])
+                                      AC_MSG_RESULT([Yes. A wrong libevent.so was linked.])
+                                      orte_check_lsf_event_conflict=yes])
+                       ],
+                       [AC_MSG_CHECKING([for libevent conflict])
+                        AC_MSG_RESULT([No. Internal Libevent is being used.])
+                        orte_check_lsf_event_conflict=na])],
+                [AC_MSG_CHECKING([for libevent conflict])
+                 AC_MSG_RESULT([No. LSF checks passed.])
+                 orte_check_lsf_event_conflict=na])
+
+          AS_IF([test "$orte_check_lsf_event_conflict" = "yes"],
+                [AC_MSG_WARN([===================================================================])
+                 AC_MSG_WARN([Conflicting libevent.so libraries detected on the system.])
+                 AC_MSG_WARN([])
+                 AC_MSG_WARN([A system installed Libevent library was detected and the Open MPI])
+                 AC_MSG_WARN([build system chose to use the 'external' component expecting to])
+                 AC_MSG_WARN([link against the Libevent in the default search path.])
```

Comment:
It may not be the default search path -- you might just want to say "in the linker search path."